source/localizable/tutorial/routes-and-templates.md

```diff
@@ -214,7 +214,7 @@ Let's update our `index.hbs` with some HTML for our home page and our links to t
 
 In addition to providing button-style links in each route of our application, we would like to provide a common banner to display both the title of our application, as well as its main pages.
 
-First, create the application by typing `ember g template application`.
+First, create the global template for the application by typing `ember g template application`.
```

Comment:
I'd rather insert the missing "template" instead, as in "create the application template". What do you think?